docs/assemblies/assembly-schema-validation-filter.adoc

```diff
@@ -0,0 +1,27 @@
+// file included in the following:
+//
+// assembly-built-in-filters.adoc
+
+[id='assembly-schema-validation-filter-{context}']
+= (Preview) Record Validation
```

Comment:
Given this title and the fact that we are renaming the file anyway should we just rename to `assembly-record-validation-filter.adoc` ?